webapi/src/test/scala/org/knora/webapi/e2e/v2/ResourcesRouteV2E2ESpec.scala

```diff
@@ -64,7 +64,7 @@ class ResourcesRouteV2E2ESpec extends E2ESpec(ResourcesRouteV2E2ESpec.config) {
     private val hamletResourceIri = new MutableTestIri
 
     // If true, writes all API responses to test data files. If false, compares the API responses to the existing test data files.
-    private val writeTestDataFiles = false
+    private val writeTestDataFiles = true
```

Comment:
Shouldn't this be `false` for the tests to actually test something?

Reply:
Could this variable be called the same in all E2E tests? Then maybe a quick search would be possible to check if it was set back correctly.

Reply:
Basically parts of this test-spec are currently turned off in `develop`.

Reply:
> Shouldn't this be `false` for the tests to actually test something?

Yes, that's a mistake.